src/lib/routes/admin-api/instance-admin.ts

```diff
@@ -129,11 +131,19 @@ class InstanceAdminController extends Controller {
     }
 
     async getStatistics(
-        req: AuthedRequest,
-        res: Response<InstanceStatsSigned>,
+        _: AuthedRequest,
+        res: Response<InstanceAdminStatsSchema>,
     ): Promise<void> {
         const instanceStats = await this.instanceStatsService.getSignedStats();
-        res.json(instanceStats);
+        const apiTokensObj = Object.fromEntries(
```

Comment:
why is it better to do it in the controller rather than the service itself?

Reply:
For visibility: slack thread with discussion: https://unleash-internal.slack.com/archives/C046LV6HH6W/p1724312294721579